get-started/system-requirements.md

```diff
@@ -4,6 +4,23 @@ title: 12. System Requirements
 description: Official TinyMCE browser support.
 keywords: browser compatibility explorer ie safari firefox chrome edge
 ---
+## Mobile Supported Platforms
+
+We're pleased to introduce TinyMCE mobile, available in version 4.7.
+
+TinyMCE mobile is designed to run on iOS Safari and Android Chrome. You'll find a streamlined interface while keeping the most common touch interactions easily to hand. We tested mobile on the following platforms:
+
+| OS        | Browser | Device       |
```

Comment:
Is this same information available on another page? If so, please refactor to a common "include" file. 

See get-started/advanced-install.md and _includes/install/cloud.md for an example of this.